src/utils/post.ts

```
@@ -193,8 +192,10 @@ export const transformApplicationDetailToJobPostingItemType = (
 };
 
 // 서버 데이터를 폼에 맞게 변환하는 함수
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const mapPostDetailDataToFormData = (postDetailData: any): JobPostingForm => {
+
+export const mapPostDetailDataToFormData = (
+  postDetailData: any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)